addons/godot-xr-tools/objects/viewport_2d_in_3d.tscn

```diff
@@ -7,13 +7,13 @@
 resource_local_to_scene = true
 size = Vector2(3, 2)
 
-[sub_resource type="ViewportTexture" id="ViewportTexture_fludi"]
+[sub_resource type="ViewportTexture" id="ViewportTexture_bfkdf"]
```

Comment:
Hmm, we need to find a way to filter out these changes, I thought the whole point of these id's was that they stay unique and I don't know why Godot keeps updating them when you clone anew. I'm going to ask about this. David I know its a PITA but any chance you can review which files only have these types of changes and remove them from the commit?

Reply:
This file does have other changes! It's this line below:

```
gui_embed_subwindows = true
```

That's necessary to allow any controls with popups to work, like the `OptionButton` that's used by the WebXR settings. I don't think there are any files in this PR that include only ID changes.